data/egtea.py

```diff
@@ -1,7 +1,8 @@
-import os
 
-from PIL import Image
+import glob
```

Comment:
Consider using `pathlib`, it will provide you with easy navigation in directories and `glob` for parsing